services/brig/test/integration/API/Provider.hs

```diff
@@ -66,8 +69,17 @@ import qualified Network.Wai.Handler.WarpTLS  as Warp
 import qualified Network.Wai.Route            as Wai
 import qualified Test.Tasty.Cannon            as WS
 
-tests :: Manager -> DB.ClientState -> Brig -> Cannon -> Galley -> IO TestTree
-tests p db b c g =
+data Config = Config
+    { privateKey   :: FilePath
+    , publicKey    :: FilePath
+    , cert         :: FilePath
+    } deriving (Show, Generic)
+
+instance FromJSON Config where
+
+tests :: Either ParseException Config -> Manager -> DB.ClientState -> Brig -> Cannon -> Galley -> IO TestTree
```

Comment:
can we avoid passing a `Either ParseException Config` into every test? That failure arising from a missing configuration entry or missing ENV variable could be checked before starting any test. Would be better to handle this in `Main`.

Reply:
yes, that would have been preferable, but I don't see how this can be resolved in main since the environment vars are pulled directly in the tests.

Would `Maybe Config` help?